spring-cloud-stream-release-train/pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.1.2.BUILD-SNAPSHOT</version>
+		<relativePath />
+	</parent>
+	<artifactId>spring-cloud-stream-release-train</artifactId>
```

Comment:
The one in spring-cloud-stream could be merged with the parent pom and all the dependencies managed for users in this one (which can then be spring-cloud-stream-dependencies again).